### PR TITLE
Bug/one team owner #314

### PR DESCRIPTION
**Pull request by va-dym-ko-val**
(1) -> (2)
image
image

(3) -> (4)
image
image

### CODE REVIEW

File: frontend/src/app/modules/settings/team/team-members/team-members.component.ts [hunk 1]
```diff
@@ -161,29 +163,37 @@ export class TeamMembersComponent extends BaseComponent implements OnInit, OnDes
...
-                    this.notificationService.showSuccessMessage('Team member`s role was successfully changed');
-                    this.reloadTeamMembers();
-                },
-                (error) => {
-                    this.notificationService.showErrorMessage(error);
-                },
-            );
+        if (newRole === Role.Owner && this.ownerId) {
+            this.notificationService.showErrorMessage('Team member with owner role should be the only');
+            this.reloadTeamMembers();
```
Comment (Ymkavi) [inline frontend/src/app/modules/settings/team/team-members/team-members.component.ts]: Do we really need to make reload request here if the action is not available?

File: frontend/src/app/modules/settings/team/team-members/team-members.component.ts [hunk 2]
```diff
@@ -161,29 +163,37 @@ export class TeamMembersComponent extends BaseComponent implements OnInit, OnDes
...
+        } else if (user.id === this.ownerId) {
+            this.notificationService.showErrorMessage('Owner role can be modified only when owner leave the team');
+            this.reloadTeamMembers();
```
Comment (Ymkavi) [inline frontend/src/app/modules/settings/team/team-members/team-members.component.ts]: Same here

**Review by Ymkavi — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/settings/team/team-members/team-members.component.ts (hunk 2 above)
Comment (va-dym-ko-val) [reply frontend/src/app/modules/settings/team/team-members/team-members.component.ts]: If I try to change role, I will see error but select has new value (only visually). We need reload members or reload page.
image

Comment (va-dym-ko-val) [reply frontend/src/app/modules/settings/team/team-members/team-members.component.ts]: For example, I can change this duplicate code to the end of method.
```suggestion
            this.reloadTeamMembers();
```

Comment (Ymkavi) [reply frontend/src/app/modules/settings/team/team-members/team-members.component.ts]: I think resetting the value in the select will be better than an api request

**Review by Ymkavi — APPROVED** [review]

File: frontend/src/app/modules/settings/team/team-members/team-members.component.ts [hunk 3]
```diff
@@ -161,14 +164,27 @@ export class TeamMembersComponent extends BaseComponent implements OnInit, OnDes
...
+        if (selectedRole === Role.Owner && this.ownerId) {
+            this.notificationService.showErrorMessage('Team member with owner role should be the only');
+
+            return;
+        }
+        if (user.id === this.ownerId) {
+            this.notificationService.showErrorMessage('Owner role can be modified only when owner leave the team');
+
+            return;
+        }
```
Comment (tatianahutii) [inline frontend/src/app/modules/settings/team/team-members/team-members.component.ts]: Why we cannot just disable dropdown?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/settings/team/team-members/team-members.component.ts (hunk 3 above)
Comment (va-dym-ko-val) [reply frontend/src/app/modules/settings/team/team-members/team-members.component.ts]: According to QA feedback, only dropdowns disabling isn`t enough

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**